### PR TITLE
Simplified enb config

### LINKED ISSUE #883: Simplify enb config

### PR DESCRIPTION
**Pull request by blond**
Resolved #883

### CODE REVIEW

File: .enb/make.js [hunk 1]
```diff
@@ -338,187 +150,127 @@ module.exports = function(config) {
...
+    );
 }
 
-function getTouchPhoneBrowsers() {
-    return [
-        'android 4',
-        'ios 6',
-        'ie 10'
-    ];
+function getBrowsers(platform) {
```
Comment (tadatuta) [inline .enb/make.js:252]: consider using `switch`

**Comment by tadatuta** [comment]
rebase needed

**Comment by blond** [comment]
@tadatuta, done!

**Pull request merged**